Wrappers/Python/test/test_Operator.py

```diff
@@ -110,7 +111,48 @@ def test_ScaledOperator(self):
         scalar = 0.5
         sid = scalar * Identity(ig)
         numpy.testing.assert_array_equal(scalar * img.as_array(), sid.direct(img).as_array())
+    
+    def test_DiagonalOperator(self):
+        print ("test_DiagonalOperator")
+        
+        M = 3
+        ig = ImageGeometry(M, M)
+        x = ig.allocate('random_int')
```

Comment:
please seed the random number generator with `numpy.random.seed(1)` or use the `seed=1` argument of `allocate`.

Also, since in #535 the type returned by `random_int` has changed, I'd suggest to use `random`.